### ISSUE TITLE
rominator425

### ISSUE THREAD

**Issue by Rominator425** [HabitRPG/habitica#3173]
the problem

about a month and a half ago my party was fighting a boss and one of us died because they hadn't logged in a long time. about 2 or 3 weeks later while they were still dead apparently the system thought it would be a good idea to have the character attack the boss despite being dead. the character did 0 damage and Vice turned around and did over 100 damage to the entire party! 

naturally I thought this was the fault of the party member. I was wrong. a couple weeks ago, while fighting the same boss, I died and to avoid having the same thing happen again I decided to stay dead by not logging in. once again, I WAS WRONG AND IT HAPPENED AGAIN. the system had my character attack the boss for 0 damage and the boss turned around and did 173 damage TO EVERY SINGLE PARTY MEMBER!. For the second time I lost a level all my gold and one of my pieces of equipment (a ski-sassin pole which is VERY EXPENSIVE! and hard to get)

the bottom line. 

the death mechanic in this game as far as boss fights go needs to be fixed immediately. twice now I have lost gold, equipment, levels and now a member of my party (they quit) as a result of this specific problem.

**Comment by meganstrickland** [comment]
Hello Rominator425! You seem to have encountered a known bug, which is that death does not remove party members from quests. It is indeed frustrating, and I assure you that it is on the list to be fixed. In the mean time, if you are feeling de-motivated by this turn of events you can restore your gold and other stats under Settings > Fix Character Values. And if you know that you or a party member is not going to be able to complete a lot of dailies, you might consider resting in the Inn. You still _take_ boss damage from other people's missed dailies when resting in the Inn, but you don't _cause_ damage to the party.

**Comment by deilann** [comment]
Just a nit-pick -- it's... not quite a bug and more that it hasn't been implemented yet. :x

**Comment by Rominator425** [comment]
Thank you Megan! :) I didn't think anyone would get back to me that quickly. I'll give it a try. just curious, does this allow us to get back equipment as well

**Comment by deilann** [comment]
What you can do is reset your gold to what you had plus the cost of the
equipment. :)

**Comment by meganstrickland** [comment]
You are quite welcome! It will not _automatically_ restore equipment, no, but assuming that you are the same class you should be able to re-purchase it. And, as deilann points out, you can refund yourself the cost of the equipment.

**Comment by Rominator425** [comment]
this is awesome because now my party members can restore themselves ( if my dead character decides to attack vice LOL )
thank you so much guys

**Comment by deilann** [comment]
No problem!

**Comment by Rominator425** [comment]
I always thought it was kinda cool to be able to stay in the challenge even after dying ( kinda like a resurrection spell)  but not if Vice is gonna do holy shit loads of damage from it lol

**Comment by deilann** [comment]
I'm actually of the opinion that dying shouldn't take you out of quests...
especially if you wouldn't get the scroll back if you failed.
